frigate/detectors/detector_type.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class DetectorTypeEnum(str, Enum):
```

Comment:
Can this enum definition be placed right in the __init__ file?

Reply:
this is moved to __init__.py now